IB Diploma/src/Domain/CommitmentGateway.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Gibbon\Module\HelpDesk\Domain;
+
+use Gibbon\Domain\Traits\TableAware;
+use Gibbon\Domain\QueryCriteria;
+use Gibbon\Domain\QueryableGateway;
+
+/**
+ * Technician Gateway
+ *
+ * @version v20
+ * @since   v20
```

Comment:
```suggestion
 * Commitment Gateway
 *
 * @version v21
 * @since   v21
```

Reply:
Good catch, I should perhaps proofread my code after copy and pasting more often